Workflow failure: master

## Workflow Failure

[master](https://github.com/pulumi/pulumi-policy-aws/blob/master/.github/workflows/master.yml) has failed. See the list of failures below:

- [2024-03-28T14:00:06.000Z](https://github.com/pulumi/pulumi-policy-aws/actions/runs/8468364707)

Error publishing the dev release of the npm package:

```
npm ERR! code ENEEDAUTH
npm ERR! need auth This command requires you to be logged in to https://registry.npmjs.org/
npm ERR! need auth You need to authorize this machine using `npm adduser`
```

--- a/.github/workflows/master.yml
+++ b/.github/workflows/master.yml
@@ -18,6 +18,7 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: ${{matrix.node-version}}
+          registry-url: https://registry.npmjs.org
       - name: Install pulumictl
         uses: jaxxstorm/action-install-gh-release@v1.2.0
         with:
@@ -60,6 +61,7 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: ${{matrix.node-version}}
+          registry-url: https://registry.npmjs.org
       - name: Ensure dependencies
         run: make ensure
       - name: Checkout Scripts Repo

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -18,6 +18,7 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: ${{matrix.node-version}}
+          registry-url: https://registry.npmjs.org
       - name: Install pulumictl
         uses: jaxxstorm/action-install-gh-release@v1.2.0
         with:
@@ -60,6 +61,7 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: ${{matrix.node-version}}
+          registry-url: https://registry.npmjs.org
       - name: Ensure dependencies
         run: make ensure
       - name: Checkout Scripts Repo

--- a/.github/workflows/run-acceptance-tests.yml
+++ b/.github/workflows/run-acceptance-tests.yml
@@ -34,6 +34,7 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: ${{matrix.node-version}}
+          registry-url: https://registry.npmjs.org
       - name: Install pulumictl
         uses: jaxxstorm/action-install-gh-release@v1.2.0
         with:
@@ -79,6 +80,7 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: ${{matrix.node-version}}
+          registry-url: https://registry.npmjs.org
       - name: Ensure dependencies
         run: make ensure
       - name: Checkout Scripts Repo

